packages/@stimulus_reflex/polyfills/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@stimulus_reflex/polyfills",
+  "version": "3.4.0-dev.4",
+  "description": "Polyfills to support StimulusReflex in older browsers",
+  "homepage": "https://docs.stimulusreflex.com/",
+  "bugs": "https://github.com/hopsoft/stimulus_reflex/issues",
+  "repository": "https://github.com/hopsoft/stimulus_reflex",
+  "license": "MIT",
+  "author": "Nathan Hopkins <natehop@gmail.com>",
+  "source": "index.js",
+  "main": "dist/polyfills.js",
+  "module": "dist/polyfills.module.js",
+  "esmodule": "dist/polyfills.modern.js",
+  "scripts": {
+    "prepare": "yarn build",
+    "prettier-standard:check": "yarn run prettier-standard --check *.js **/*.js",
+    "prettier-standard:format": "yarn run prettier-standard *.js **/*.js",
+    "clean": "rimraf dist",
+    "build": "microbundle --target browser --format modern,es,cjs --no-strict",
+    "dev": "microbundle watch --target browser --format modern,es,cjs --no-strict"
+  },
+  "dependencies": {
+    "@stimulus/polyfills": "^1.1.1",
+    "@webcomponents/template": "^1.4.4",
+    "core-js": "^3.4.0",
+    "formdata-polyfill": "^3.0.20"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
```

Comment:
Should we perhaps add a sideEffects declaration as well?

Something like: 

```json
sideEffects: [
  "./index.js",
  "./polyfills/**/*.js"
]
```

This helps Webpack optimize treeshaking. Here's the link to the docs.

https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free

Reply:
Thank you @ParamagicDev, good catch! I think it make sense to declare them as side effects. I'll update the PR!

Do you know if there is a way to test that the optimization really kicked in?

Reply:
Appears you may just have to link and build and see what happens.

This was the best I could find:

https://github.com/webpack/webpack/tree/master/examples/side-effects